docs/dependency-updates.md

```diff
@@ -0,0 +1,164 @@
+When dependencies release new versions, we want to make sure they don't break
+Relay when we upgrade. This document describes what to look for when an upgrade
+comes in.
+
+## Things to know
+
+Typically, we receive update notifications from
+[Dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/about-dependabot-alerts),
+which periodically checks for new versions of dependencies, and then submits
+pull requests to update them.
+
+Some terms to be aware of:
+
+- **Checks**: at the bottom of a Pull Request's page, GitHub will list the
+  outcomes of Continuous Integration runs. It calls these "checks".
+- **CI results**: to view the results of a CI run (e.g. `test_frontend`), click
+  the "Details" link next to the respective Check.
+- **Preview deployment**: for
+  every PR, a copy of the frontend with that PR's code is deployed to Netlify. You
+  can view that deployment by clicking the "Details" link next to the
+  `netlify/fx-relay-demo/deploy-preview` Check.
+
+All dependency updates involve verifying that all Checks are successful,
+investigating the results of failing CI runs, and a cursory inspection of the
+changelog (if any) for anything that might be relevant to Relay, in particular
+if a dependency's [major version](https://semver.org/) is bumped, or if a
+dependency doesn't use Semantic Versioning.
+
+If you're unsure whether anything looks as expected, try to replicate what
+you're doing on the `main` branch. If the results differ, there's probably an
+issue with the update.
+**Make sure to do a fresh install of the dependencies when switching branches!**
+
+You can line up multiple Dependabot PRs for merging by adding the comment
+`@dependabot merge`. Dependabot will then automatically keep the PR up-to-date
+with changes in `main`, and merge it if the Checks continue to run successfully.
+Alternatively, you can use the CLI tool [`pmac`](https://github.com/willkg/paul-mclendahand).
+
+## npm
+
+We have two npm projects in this repository: the website frontend (identified by
+`/frontend/package.json`), and the root project from which our end-to-end tests
+are run (identified by `/package.json`). The former is defined as a
```

Comment:
question (non-blocking): should we move the E2E test `package.json` to `e2e-tests/package.json` ?

Reply:
Maybe! :shrug: It's not necessarily part of the frontend (one end of the end-to-end is the backend :) ), but it is pretty confusing to have two `package.json`s, I'm not always 100% sure where to run which command, and it sometimes confuses our tooling (e.g. Dependabot) as well, so I'd be in favour of that.

One downside is that you'd no longer be able to run `npm install` in the root, but I think npm gives a clear error message there, so that's probably not a biggie.